Lib/asyncio/base_events.py

```diff
@@ -107,7 +107,7 @@ def _ipaddr_info(host, port, family, type, proto):
             host is None:
         return None
 
-    if type == socket.SOCK_STREAM:
+    if _is_stream_socket(type):
```

Comment:
This change is ok. But please fix also "type == socket.SOCK_DGRAM" below, fix test_ipaddr_info(), and please also write a test for non-blocking DGRAM socket.

Reply:
Ping @1st1